docs/_documentations/troubleshooting.md

```diff
@@ -390,6 +390,33 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Codewind version: 0.6.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1061
+-->
+## Appsody binds fail
+
+If an Appsody repository is already added in your local Appsody CLI, the Appsody bind might fail.
+
+These steps create the issue:
+1. The Appsody repository was added to your local Appsody CLI from your IDE by right-clicking **Projects (Local)** and then clicking **Manage Template Sources**. The same Appsody repository was then added to Codewind from your IDE.
```

Comment:
1. The Appsody repository was added to your local Appsody CLI.  The same Appsody repository was then added to Codewind from your IDE by right-clicking **Projects (Local)** and then clicking **Manage Template Sources**.

Just a minor rearrangement then we're all good

Reply:
Done! Thanks!